input/pagecontent/content-transitions.md

```diff
@@ -107,6 +107,7 @@
 <tr><td> <a href='ValueSet-ValueSet-input-race-and-ethnicity-person-vr.html'>ValueSetInputRaceAndEthnicityPersonVitalRecords</a> </td><td> New </td><td> Incorporated standard vital records approach to race & ethnicity (generalized to support all vital records use cases) </td></tr>
```

Comment:
On lines 82/83 I would check that links work as expected with the auxiliary state id consolidation.